Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -192,33 +197,41 @@ public void ClimbCheck()
 		{
 			if (climbCheckTimer > climbCheckTime)
 			{
-				RaycastHit hit;
-				Physics.SphereCast( new Ray( transform.position, model.forward ), climbCheckRadius, out hit, climbCheckDistance, climbLayer );
-				if ( hit.transform )
-				{
-					StartClimbing( hit.collider );
-				}
-				else
+				if (_actorStats.CanUseStamina())
```

Comment:
I moved it to where it is so that it would only be checked when the climb check timer goes. DO you think it makes more sense with isGrabbing?